lib/sfn-parameters/stacks.rb

```diff
@@ -10,7 +10,7 @@ class ParametersStacks < ParametersInfrastructure
       # @param stack_name [String]
       # @return [Smash]
       def load_file_for(stack_name)
-        root_path = config.fetch(:sfn_parameters, :directory, 'stacks')
+        root_path = config.fetch(:sfn_parameters, :directory, 'stacks', '**')
```

Comment:
This modification would change the configuration location for the directory path from:

```ruby
{sfn_parameters: {directory: 'PATH'}}
```

with `"stacks"` being the default directory to:

```ruby
{sfn_parameters: {directory: {stacks: 'PATH'}}}
```

with `"*"` being the default value. The `Smash#fetch` method works much like the `Hash#dig` method (just before `#dig` was available). Were you thinking of something like:

```ruby
root_path = config.fetch(:sfn_parameters, :directory, "stacks#{File::DELIMITER}*")
```